src/main/java/com/gengo/client/payloads/Attachment.java

```diff
@@ -0,0 +1,110 @@
+package com.gengo.client.payloads;
+
+import org.json.JSONObject;
+import org.json.JSONException;
+
+import com.gengo.client.exceptions.GengoException;
+
+
+public class Attachment extends Payload
+{
+    protected JSONObject data;
+
+    protected static final String URL = "url";
+    protected static final String FILENAME = "filename";
+    protected static final String MIMETYPE = "mime_type";
```

Comment:
These variables should be `private` rather than `protected` unless this class is expected to be extended by other classes.